views/createBooking/newBooking.njk

```diff
@@ -59,6 +59,21 @@
                             }
                         ]
                     }) }}
+                    {% if courts.length === 1 %}
+                    {{ govukSummaryList({
```

Comment:
very meh.. but new blocks: if/for/else etc.. should probably increase the indentation of its contents

Reply:
Noted for the future